app/assets/stylesheets/components/_btn.scss

```diff
@@ -90,22 +11,23 @@
   }
 }
 
-.btn-disabled {
-  background-color: $gray-light;
-  color: $gray-dark;
-  cursor: not-allowed;
-}
-
 .btn-account-action {
```

Comment:
since this is now entirely our custom style, should we name it differently to differentiate from BassCSS? Like `button--account-action` or something?

Reply:
We could, yeah. Long-term, I'm not sure if or how these buttons should fit into our inventory of buttons, and as such if it's worth much effort in formalizing. I'm failing to recall it now, but I remember some discussion during the original button design work in LG-3907 as to whether we wanted these buttons to be in the design system, and decided against it in-part because of some of the issues they present (layout issues like in #4454, accessibility concerns like [WCAG SC 2.5.5](https://www.w3.org/WAI/WCAG22/Understanding/target-size.html)).

For now, I think this is a good call-out as a piece of work I could extract from this pull request, which is: Refactor the account action button as being independent from BassCSS, optionally renamed.

Reply:
> For now, I think this is a good call-out as a piece of work I could extract from this pull request, which is: Refactor the account action button as being independent from BassCSS, optionally renamed.

For posterity, this happened at #4840.